src/tests/Common/dirs.proj

```diff
@@ -18,9 +18,10 @@
     </ItemGroup>
 
     <ItemGroup>
-      <AllProjects Include="$(TestRoot)**\*.csproj" Exclude="@(DisabledProjects)" />
+      <!-- <AllProjects Include="$(TestRoot)**\*.csproj" Exclude="@(DisabledProjects)" />
```

Comment:
I assume this is just local instrumentation not to be merged in, otherwise we need to change this to android-specific test filtering unless you intend to turn off most of CoreCLR testing ;-).

Reply:
You are right, it is just for faster local development.